test/recipes/aaveV3/boost-with-fl-carry-debt-action.js

```diff
@@ -0,0 +1,256 @@
+const { expect } = require('chai');
+const hre = require('hardhat');
+const dfs = require('@defisaver/sdk');
+const {
+    getProxy,
+    WETH_ADDRESS,
+    LUSD_ADDR,
+    takeSnapshot,
+    revertToSnapshot,
+    setBalance,
+    addrs,
+    getNetwork,
+    redeploy,
+    nullAddress,
+    formatExchangeObj,
+    UNISWAP_WRAPPER,
+    getAddrFromRegistry,
+} = require('../../utils');
+
+const { aaveV3Supply, aaveV3Borrow, executeAction } = require('../../actions');
+
+const {
+    VARIABLE_RATE,
+    AAVE_NO_DEBT_MODE,
+    WETH_ASSET_ID_IN_AAVE_V3_MARKET,
+    LUSD_ASSET_ID_IN_AAVE_V3_MARKET,
+} = require('../../utils-aave');
+
+const aaveV3BoostWithNewFL = async () => {
+    describe('Aave V3 AaveV3 Boost Tests With New FL', () => {
```

Comment:
`AaveV3 Boost Tests with Carry Debt FL`, or something like that. NewFL not good description as we might later on add a newer one :D